api/src/pcapi/routes/backoffice/templates/collective_offer/details.html

```diff
@@ -79,6 +95,13 @@ <h2 class="card-title mb-3 text-primary">{{ collective_offer.name }}</h2>
                   <span class="fw-bold">Lieu :</span> {{ links.build_venue_name_to_details_link(collective_offer.venue) }}
                 </p>
               </div>
+              {% if collective_offer.template %}
+                <div class="fs-6">
+                  <p class="mb-1">
+                    <span class="fw-bold">Offre vitrine lié :</span> {{ links.build_offer_name_to_pc_pro_link(collective_offer.template) }}
```

Comment:
Il est demandé dans le ticket « Lien (BO) vers l’offre collective vitrine associée (si elle existe) », tu as mis un lien vers PC Pro ; à corriger.
Ainsi qu'une faute d'accord : 
```suggestion
                    <span class="fw-bold">Offre vitrine liée :</span> {{ links.build_offer_name_to_pc_pro_link(collective_offer.template) }}
```

Reply:
Ne pouvant créer de lien ( celui ci faisant partie de l'autre ticket, j'ai ajouté un TODO apres avoir créer la macro pour modifier le lien temporaire )